<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1400</width>
    <height>754</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1400</width>
    <height>725</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-color: Black;</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <property name="leftMargin">
     <number>200</number>
    </property>
    <property name="topMargin">
     <number>100</number>
    </property>
    <property name="rightMargin">
     <number>200</number>
    </property>
    <property name="bottomMargin">
     <number>100</number>
    </property>
    <item>
     <widget class="QFrame" name="mainPannel">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>900</width>
        <height>500</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #f2f2f2;
border-radius: 12px;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="leftMargin">
        <number>9</number>
       </property>
       <item>
        <widget class="QWidget" name="sidebar" native="true">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>260</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">#sidebar {
	background-color: white;
}

#sidebar QPushButton {
    background-color: #ffffff;
    color: #1E3A5F;            /* blu scuro */
    border: 2px solid #1E90FF; /* blu acceso */
    border-radius: 18px;
    padding: 6px 18px;
    font-size: 11pt;
    font-weight: 500;
    text-align: center;
}

/* Hover */
#sidebar QPushButton:hover {
    background-color: #1E90FF;
    color: #ffffff;
}

/* Pulsante premuto */
#sidebar QPushButton:pressed {
    background-color: #1564b3;
    border-color: #1564b3;
}

#sidebar QPushButton:checked {
    background-color: #1E90FF;
    color: white;
}
</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="spacing">
           <number>20</number>
          </property>
          <property name="leftMargin">
           <number>30</number>
          </property>
          <property name="topMargin">
           <number>30</number>
          </property>
          <property name="rightMargin">
           <number>30</number>
          </property>
          <property name="bottomMargin">
           <number>30</number>
          </property>
          <item>
           <widget class="QLabel" name="lblLogo">
            <property name="styleSheet">
             <string notr="true">border-radius: 60px;     /* met√† della dimensione della label */
background-color: white;
padding: 5px;
</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="scaledContents">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnAreaClienti">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>Area Clienti</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnAreaDipendenti">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>Area Dipendenti</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnAreaServizi">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>Area Servizi</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnAreaInterventi">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>Area Interventi</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnCalendario">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>Calendario</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QStackedWidget" name="stackedContent">
         <property name="minimumSize">
          <size>
           <width>600</width>
           <height>0</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: #e5e5e5;</string>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="pageClienti">
          <widget class="QWidget" name="">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>10</y>
             <width>551</width>
             <height>382</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,1,0">
            <item>
             <widget class="QLabel" name="lblClientiTitle">
              <property name="font">
               <font>
                <pointsize>16</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color: Black;</string>
              </property>
              <property name="text">
               <string>Gestione Clienti</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignTop</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QWidget" name="widget" native="true">
              <property name="styleSheet">
               <string notr="true">QPushButton{
	color : black;
}</string>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout">
               <property name="spacing">
                <number>15</number>
               </property>
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QPushButton" name="btnClienteAggiungi">
                 <property name="text">
                  <string>Aggiungi</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="btnClienteModifica">
                 <property name="text">
                  <string>Modifica</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="btnClienteElimina">
                 <property name="text">
                  <string>Elimina</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QTableWidget" name="tableClienti">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>300</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QTableWidget {
    background-color: #ffffff;
    color: #222222;                  /* testo nero */
    gridline-color: #dddddd;
    selection-background-color: #1E90FF;
    selection-color: #ffffff;
}

QHeaderView::section {
    background-color: #f5f5f5;       /* grigio chiaro */
    color: #222222;                  /* testo nero */
    font-weight: bold;
    padding: 4px;
    border: none;
    border-bottom: 1px solid #dddddd;
}

</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
         <widget class="QWidget" name="pageDipendenti">
          <widget class="QLabel" name="label_11">
           <property name="geometry">
            <rect>
             <x>230</x>
             <y>160</y>
             <width>131</width>
             <height>16</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">color:black;</string>
           </property>
           <property name="text">
            <string>pageDipendenti</string>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="pageServizi">
          <widget class="QLabel" name="label_7">
           <property name="geometry">
            <rect>
             <x>260</x>
             <y>150</y>
             <width>81</width>
             <height>16</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">color:black;</string>
           </property>
           <property name="text">
            <string>pageServizi</string>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="pageInterventi">
          <widget class="QLabel" name="label_9">
           <property name="geometry">
            <rect>
             <x>270</x>
             <y>170</y>
             <width>91</width>
             <height>16</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">color:black;</string>
           </property>
           <property name="text">
            <string>pageInterventi</string>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="pageCalendario">
          <widget class="QLabel" name="label_10">
           <property name="geometry">
            <rect>
             <x>240</x>
             <y>220</y>
             <width>91</width>
             <height>16</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">color:black;</string>
           </property>
           <property name="text">
            <string>pageCalendario</string>
           </property>
          </widget>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1400</width>
     <height>33</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
